src/types/index.ts

```diff
@@ -741,6 +749,12 @@ export interface TransferFormModel extends IFormModel {
   payload: string;
 }
 
+export type TransferArgs = {
```

Comment:
I know there is not much of a difference between type and interface but in most cases our app uses interfaces prefixed with "I". Maybe this one could follow this to be more consistent?